content/intro-to-custom-on-chain-programs.md

```diff
@@ -189,16 +187,16 @@ Copy the transaction signature. Open a browser and go to [https://explorer.solan
 ![Screenshot of Solana Explorer with logs from calling the Ping program](../assets/solana-explorer-ping-result.png)
 
 Scroll around the Explorer and look at what you're seeing:
- - The **Account Input(s)** will include: 
+- The **Account Input(s)** will include:
   - The address of your payer - being debited 5000 lamports for the transaction
   - The program address for the ping program
   - The data address for the ping program
- - The **Instruction** section will contain a single instruction with no data - the ping program is a pretty simple program, so it doesn't need any data.
- - The **Program Instruction Logs** shows the logs from the ping program.  
+- The **Instruction** section will contain a single instruction with no data - the ping program is a pretty simple program, so it doesn't need any data.
+- The **Program Instruction Logs** shows the logs from the ping program.
 
 [//]: # "TODO: these would make a good question-and-answer interactive once we have this content hosted on solana.com, and can support adding more interactive content easily."
 
-If you want to make it easier to look at Solana Explorer for transactions in the future, simply change your `console.log` to the following:
+If you want to make it easier to look at the Solana Explorer for transactions in the future, simply change your `console.log` to the following:
```

Comment:
Solana Explorer isn't actually mentioned in the style guide but maybe this is a reason it should be.

I made the change for consistency, the six other times it is mentioned on the page it is preceded with 'the', so it being missing was clearly the odd one out here.

I agree in principle that as a service the definite article wouldn't be expected however as a noun it could be, 'the Statue of Liberty' is an example. A definitive explorer on Solana would clearly be referred to as 'the Solana explorer' so having 'explorer' as part of the name is what has probably resulted in the inconsistency here.

I will revert this change and remove 'the' from all the other existing instances.